test-engineering/contract-tests/volumes/contile/adm_settings.json

```diff
@@ -1,104 +1,186 @@
 {
-    "Example COM": {
-        "advertiser_urls": [
-            {
-                "host": "www.example.com"
-            },
-            {
-                "host": "www.example.co.uk"
-            },
-            {
-                "host": "www.example.it"
-            },
-            {
-                "host": "www.example.com.au"
-            },
-            {
-                "host": "www.example.ca"
-            },
-            {
-                "host": "www.example.de"
-            },
-            {
-                "host": "www.example.fr"
-            },
-            {
-                "host": "www.example.es"
-            },
-            {
-                "host": "www.example.in"
-            },
-            {
-                "host": "www.example.com.mx"
-            }
-        ],
-        "click_hosts": [],
-        "impression_hosts": [],
-        "include_regions": [],
-        "position": 1
-    },
-    "Example ORG": {
-        "advertiser_urls": [
-            {
-                "host": "www.example.org"
-            }
-        ],
-        "click_hosts": [
-            "example.org"
-        ],
-        "impression_hosts": [
-            "example.org"
-        ],
-        "include_regions": [],
-        "position": 1
-    },
-    "DunBroch": {
-        "advertiser_urls": [
-            {
-                "host": "www.dunbroch.co.uk",
-                "paths": [
-                    {
-                        "value": "/gb_desktop_macos/",
-                        "matching": "prefix"
-                    },
-                    {
-                        "value": "/gb_desktop_linux",
-                        "matching": "exact"
-                    }
-                ]
-            }
-        ],
-        "click_hosts": [
-            "dunbroch.co.uk"
-        ],
-        "impression_hosts": [
-            "dunbroch.co.uk"
-        ],
-        "include_regions": [
-            "GB"
-        ],
-        "position": 2
-    },
-    "DEFAULT": {
-        "advertiser_urls": [],
-        "click_hosts": [
-            "example.com"
-        ],
-        "impression_hosts": [
-            "example.com"
-        ],
-        "include_regions": [
-            "US",
-            "GB",
-            "IT",
-            "AU",
-            "CA",
-            "DE",
-            "FR",
-            "ES",
-            "IN",
-            "MX",
-            "CN"
-        ]
-    }
+  "Example COM": {
```

Comment:
Sorry I should have suggested this much earlier, but how about adding another level of nesting to these json settings? It would make adding extra buckets of data later easier if needed.

```json
{
    "adm_advertisers": {
        "Example COM": {
             ...
        }
    },
    "last_modified": "2022-10-17 21:50:00 UTC"
}
```

E.g. a timestamp from shepherd (granted we could also use GCS's file timestamps for that)

Reply:
Let's make that a future feature, since we'll have to modify Shepherd output as well.